### DISCUSSION TITLE
How to remove a node that is not ready

### DISCUSSION THREAD

**Discussion by fabnavigator** [zwave-js/node-red-contrib-zwave-js#223] [Q&A] [answered]
Hi @marcus-j-davies,

I was adding a tilt sensor node, and ended up with it added twice. Now I have node 47 and 48 for the same device. Node 47 needs to be removed. It comes up as not ready and seems to be preventing the network initialization from completing.

![Untitled](https://user-images.githubusercontent.com/16625904/173404743-c48b38cd-dbe4-4b91-8a1b-8305d99bdcd2.png)

**Comment by marcus-j-davies** [comment, marked as answer]
Hi @fabnavigator 
this was a fix in V7.


if you Shift + Click
you get these options - one of them, is remove failed node.



![image](https://user-images.githubusercontent.com/55892693/173406088-07e595ce-cdfd-4c90-9f78-0e119dbb47a1.png)


Shift + Click reveals options without the need for the  node being selected.

**Reply by fabnavigator** [reply]
Thank you.

Any chance you would be willing to add a note about that to the dialog that said the node is not ready?

i.e., "Node not ready. Shift-click for additional options."

**Reply by marcus-j-davies** [reply]
Yeah no worries!
I will re-word the prompt during the next update 👍

**Reply by fabnavigator** [reply]
Thank you.
